crates/holochain/Cargo.toml

```diff
@@ -110,6 +110,7 @@ unwrap_to = { version = "0.1.0", optional = false }
 arbitrary = { version = "1.0", features = ["derive"] }
 
 [build-dependencies]
+hdk = { version = "0.0.145", path = "../hdk"}
```

Comment:
is this dependent on the `hdk` feature or is it unconditional? (i.e., how do build-dependencies work regarding features?)

Reply:
since optional is not specified, hdk is required in the build context. But it will still only be optionally included in actual lib or bin builds